Tasks/SqlAzureDacpacDeployment/task.json

```diff
@@ -36,7 +36,7 @@
 	"inputs": [
 		{
 			"name": "ConnectedServiceName",
-			"type": "connectedService:Azure",
+			"type": "connectedService:Azure:Certificate,UsernamePassword",
```

Comment:
Thanks. As discussed you would want to track a separate story for enabling support for ARM. Since the current Task supports only RDFE, I am fine with the filtering. You would want to update documentation that this task is only for RDFE.